framework/contrib/ice_updater/include/LibcurlUtils.h

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
```

Comment:
I wonder if this stuff, that's actually agnostic of ICE, should go in its own `contrib` directory... we could use it in other places...